Draft: npm, github, & repo activity

Rossb0b opened the pull request:
related to issue https://github.com/NodeSecure/authors/issues/9

need a new release next to this pr https://github.com/NodeSecure/github/pull/87

Rossb0b committed:
chore: install dayjs

Rossb0b committed:
feat: activity check information for npm, github, repo. If rate-limited, values as null

Rossb0b committed:
test: added related data to audited author

Rossb0b committed:
docs: updated data for authors

Rossb0b committed:
fix(index.d.ts): updated type related to activity data & cleaned names

Kawacrepe reviewed (approved):
Lgtm, will wait for the @nodesecure/github release

snyk-bot committed:
fix: upgrade @myunisoft/httpie from 1.4.1 to 1.9.1 (#16)

Snyk has created this PR to upgrade @myunisoft/httpie from 1.4.1 to 1.9.1.

See this package in npm:
https://www.npmjs.com/package/@myunisoft/httpie

See this project in Snyk:
https://app.snyk.io/org/nodesecure/project/618fe0d6-b3e5-4dbc-840c-25b96627318e?utm_source=github&utm_medium=referral&page=upgrade-pr

Kawacrepe committed:
fix: handle invalid input data (#19)

Kawacrepe committed:
chore(security): add security.md

Kawacrepe committed:
1.0.1

Kawacrepe committed:
refacto: some improvements (#20)

* refacto: some improvements

* fix: review

Kawacrepe committed:
1.0.2

fraxken committed:
refactor(SECURITY.md): Updating security policy

fraxken committed:
chore(package): update devDependencies

fabnguess committed:
refactor(README.md): Upating badge (#22)

1 - Update badge style 
2 - Adding the OSSF badge

fraxken reviewed (changes requested):
Waiting for the code from github to be merged here! Thanks <3

fabnguess committed:
feat(dependabot.yml): add dependabot yml configuration (#23)

dependabot[bot] committed:
chore(deps): bump actions/setup-node from 2 to 3 (#27)

Bumps [actions/setup-node](https://github.com/actions/setup-node) from 2 to 3.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v2...v3)

---
updated-dependencies:
- dependency-name: actions/setup-node
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump github/codeql-action from 2.1.27 to 2.1.39 (#25)

Bumps [github/codeql-action](https://github.com/github/codeql-action) from 2.1.27 to 2.1.39.
- [Release notes](https://github.com/github/codeql-action/releases)
- [Changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md)
- [Commits](https://github.com/github/codeql-action/compare/807578363a7869ca324a79039e6db9c843e0e100...a34ca99b4610d924e04c68db79e503e1f79f9f02)

---
updated-dependencies:
- dependency-name: github/codeql-action
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump ossf/scorecard-action from 2.0.6 to 2.1.2 (#24)

Bumps [ossf/scorecard-action](https://github.com/ossf/scorecard-action) from 2.0.6 to 2.1.2.
- [Release notes](https://github.com/ossf/scorecard-action/releases)
- [Changelog](https://github.com/ossf/scorecard-action/blob/main/RELEASE.md)
- [Commits](https://github.com/ossf/scorecard-action/compare/99c53751e09b9529366343771cc321ec74e9bd3d...e38b1902ae4f44df626f11ba0734b14fb91f8f86)

---
updated-dependencies:
- dependency-name: ossf/scorecard-action
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump actions/checkout from 2 to 3 (#28)

Bumps [actions/checkout](https://github.com/actions/checkout) from 2 to 3.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v2...v3)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump actions/upload-artifact from 3.1.0 to 3.1.2 (#26)

Bumps [actions/upload-artifact](https://github.com/actions/upload-artifact) from 3.1.0 to 3.1.2.
- [Release notes](https://github.com/actions/upload-artifact/releases)
- [Commits](https://github.com/actions/upload-artifact/compare/3cea5372237819ed00197afe530f5a7ea3e805c8...0b7f8abb1508181956e8e162db84b466c27e18ce)

---
updated-dependencies:
- dependency-name: actions/upload-artifact
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

step-security-bot committed:
[StepSecurity] Apply security best practices (#29)

Signed-off-by: StepSecurity Bot <bot@stepsecurity.io>

Signed-off-by: StepSecurity Bot <bot@stepsecurity.io>
